exchanges/huobi/huobi_websocket.go

```diff
@@ -56,6 +60,25 @@ const (
 	rateLimit  = 20
 )
 
+var defaultSubscriptions = subscription.List{
+	{Enabled: true, Asset: asset.Spot, Channel: subscription.TickerChannel},
+	{Enabled: true, Asset: asset.Spot, Channel: subscription.CandlesChannel, Interval: kline.OneMin},
+	{Enabled: true, Asset: asset.Spot, Channel: subscription.OrderbookChannel, Levels: 0}, // Aggregation Levels; 0 is no depth aggregation
```

Comment:
Cannot be customisable to the user?
Your readme says:
>- Configure Level: 1-5 for depth aggregation

Config:
```
  {
      "enabled": true,
      "channel": "orderbook",
      "asset": "spot"
     },
```

Reply:
Did you see the line above in the README?
```
Default Public Subscriptions:
- Ticker
- Candles ( Interval: 1min )
- Orderbook ( Level: 0 - No aggregation )
  - Configure Level: 1-5 for depth aggregation
- Trades
```

Since level 0 is "don't aggregate", which is basically normal, it feels okay that it's omitted from the config.

Fixed d7896584d

Is this acceptable?

Reply:
I did see the level, but I wanted a JSON example to make it clear what's required for a user. Thanks for the update